src/get-started/java-binding.md

```diff
@@ -625,8 +625,7 @@ pass `java.library.path` system property to JVM:
 -Djava.library.path=$EXONUM_JAVA/lib/native
 ```
 
-`$EXONUM_JAVA` environment variable should point at installation location,
-as specified in [How to Run a Service section](#how-to-run-a-service).
+`$EXONUM_JAVA` environment variable should point at installation location.
```

Comment:
Some of these have been resolved in the PR properly, I'll resolve the conflicts.